### PR TITLE
Add Turkish Translations

### PR DESCRIPTION
**Pull request by neondev27**
Added path to Turkish translation

### CODE REVIEW

**Comment by neondev27** [comment]
That means i need to repull request with your suggestions?

**Comment by neondev27** [comment]
I did changes you said @Geokureli.

File: CONTRIBUTING_TR.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+HaxeFlixel'e katkıda bulunduğunuz için teşekkürler!
+## Hata bildirme
+
+Hata bildirdiğiniz zaman, size sunulan [sorun şablonu](https://github.com/HaxeFlixel/flixel/blob/dev/.github/ISSUE_TEMPLATE.md)nu basit bir biçimde doldurun.
```
Comment (Geokureli) [inline CONTRIBUTING_TR.md:4]:
```suggestion
Hata bildirdiğiniz zaman, size sunulan [sorun şablonu](https://github.com/HaxeFlixel/flixel/blob/dev/.github/ISSUE_TEMPLATE.md) nu basit bir biçimde doldurun.
```
all throughout the new files, I see spaces after links have been removed, this makes it look odd, like in the image below
Screenshot 2023-06-02 at 4 21 45 PM

File: CONTRIBUTING_TR.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+HaxeFlixel'e katkıda bulunduğunuz için teşekkürler!
```
Comment (Geokureli) [inline CONTRIBUTING_TR.md:1]: this file should should be in the .github folder with the original CONTRIBUTING.md

File: README_TR.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## Bağlantılar
+
+Burada HaxeFlixel ile başlayabilmeniz için bazı önemli bağlantılar var:
+
+- [Başlarken](https://haxeflixel.com/documentation/getting-started/)
+- [Dökümantasyon](https://haxeflixel.com/documentation)
+- [API](https://api.haxeflixel.com/)
+- [Topluluk](https://haxeflixel.com/documentation/community/)
+
+Eğer kod katkısında bulunmak veya hata bildirmek istiyorsanız, lütfen [CONTRIBUTING.md](https://github.com/neondev27/flixel/blob/dev/.github/CONTRIBUTING_TR.md)'mizi inceleyin.
```
Comment (Geokureli) [inline README_TR.md:23]: add space and change link from neondev27 to haxeflixel's .github version
```suggestion
Eğer kod katkısında bulunmak veya hata bildirmek istiyorsanız, lütfen [CONTRIBUTING.md](https://github.com/HaxeFlixel/flixel/blob/dev/.github/CONTRIBUTING_TR.md) 'mizi inceleyin.
```

File: README_TR.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Platformlar
+
+[Haxe](https://haxe.org/) and [OpenFL](http://www.openfl.org/) üzerinde yapılması sayesinde, HaxeFlixel _doğal olarak_ aşağıdaki platformları destekler:
+
+![](images/platforms.png)
+
+## Hakkında
+
+HaxeFlixel, [Adam “Atomic” Saltsman](http://www.adamatomic.com/) tarafından yapılan [Flixel'in ActionScript3 versiyonu](https://github.com/AdamAtomic/flixel)ndan köklerini almaktadır. 2011'de [Alexander Hohlov](https://github.com/beeblerox) tarafından AS3 kod tabanının ve Richard Davey'in [Flixel Güç Araçları](http://www.photonstorm.com/flixel-power-tools)nın Haxe portu olarak başladı.
```
Comment (Geokureli) [inline README_TR.md:33]:
```suggestion
HaxeFlixel, [Adam “Atomic” Saltsman](http://www.adamatomic.com/) tarafından yapılan [Flixel'in ActionScript3 versiyonu](https://github.com/AdamAtomic/flixel) ndan köklerini almaktadır. 2011'de [Alexander Hohlov](https://github.com/beeblerox) tarafından AS3 kod tabanının ve Richard Davey'in [Flixel Güç Araçları](http://www.photonstorm.com/flixel-power-tools) nın Haxe portu olarak başladı.

```

File: README_TR.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+HaxeFlixel'in bugünkü sürümü, [çekirdek ekibin](https://github.com/orgs/HaxeFlixel/people) ve [100'ün üzerinde katkıda bulunanları](https://github.com/HaxeFlixel/flixel/graphs/contributors) sayesinde bugünkü halini aldı. Yalnızca çekirdek motorunda değil birçok önemli gelişme ve yeni özellik görmenin yanı sıra, daha zengin bir ekosistem de ek kütüphanelerle ve [80'den fazla demo projesi](https://github.com/HaxeFlixel/flixel-demos)yle öğrenmeye olanak sağlamaktadır.
```
Comment (Geokureli) [inline README_TR.md:35]:
```suggestion
HaxeFlixel'in bugünkü sürümü, [çekirdek ekibin](https://github.com/orgs/HaxeFlixel/people) ve [100'ün üzerinde katkıda bulunanları](https://github.com/HaxeFlixel/flixel/graphs/contributors) sayesinde bugünkü halini aldı. Yalnızca çekirdek motorunda değil birçok önemli gelişme ve yeni özellik görmenin yanı sıra, daha zengin bir ekosistem de ek kütüphanelerle ve [80'den fazla demo projesi](https://github.com/HaxeFlixel/flixel-demos) yle öğrenmeye olanak sağlamaktadır.

```

File: README_TR.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+[![](https://raw.github.com/HaxeFlixel/haxeflixel.com/master/src/files/images/flixel-logos/HaxeFlixel.png)](http://haxeflixel.com/)
+
+[flixel](https://github.com/HaxeFlixel/flixel) | [eklentiler](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demolar](https://github.com/HaxeFlixel/flixel-demos) | [araçlar](https://github.com/HaxeFlixel/flixel-tools) | [şablonlar](https://github.com/HaxeFlixel/flixel-templates) | [dökümanlar](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com)
```
Comment (Geokureli) [inline README_TR.md:3]: optional: link back to english?
```suggestion
[flixel](https://github.com/HaxeFlixel/flixel) | [eklentiler](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demolar](https://github.com/HaxeFlixel/flixel-demos) | [araçlar](https://github.com/HaxeFlixel/flixel-tools) | [şablonlar](https://github.com/HaxeFlixel/flixel-templates) | [dökümanlar](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [english](https://github.com/HaxeFlixel/flixel/blob/dev/README.md)
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [![](https://raw.github.com/HaxeFlixel/haxeflixel.com/master/src/files/images/flixel-logos/HaxeFlixel.png)](http://haxeflixel.com/)
 
-[flixel](https://github.com/HaxeFlixel/flixel) | [addons](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demos](https://github.com/HaxeFlixel/flixel-demos) | [tools](https://github.com/HaxeFlixel/flixel-tools) | [templates](https://github.com/HaxeFlixel/flixel-templates) | [docs](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com)
+[flixel](https://github.com/HaxeFlixel/flixel) | [addons](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demos](https://github.com/HaxeFlixel/flixel-demos) | [tools](https://github.com/HaxeFlixel/flixel-tools) | [templates](https://github.com/HaxeFlixel/flixel-templates) | [docs](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [turkish](https://github.com/neondev27/flixel/blob/dev/README_TR.md)
```
Comment (Geokureli) [inline README.md:3]: this still links to the external fork
```suggestion
[flixel](https://github.com/HaxeFlixel/flixel) | [addons](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demos](https://github.com/HaxeFlixel/flixel-demos) | [tools](https://github.com/HaxeFlixel/flixel-tools) | [templates](https://github.com/HaxeFlixel/flixel-templates) | [docs](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [turkish](https://github.com/HaxeFlixel/flixel/blob/dev/README_TR.md)
```

**Review by Geokureli — CHANGES_REQUESTED** [review]
1. some formatting issues
2. link to haxeflixel pages
3. I think CONTRIBUTING.md should have a link to turkish and CONTRIBUTING_TR.md should have a link to english as well

**Comment by neondev27** [comment]
It is final look of the things. I wish it helped, @Geokureli

**Comment by neondev27** [comment]
If anything else needed, call.me from here or @ NeoDev27 in Twitter

**Review by Geokureli — APPROVED** [review]

**Comment by Geokureli** [comment]
Thanks!

**Pull request merged**